src/nvim/ex_docmd.c

```diff
@@ -3275,7 +3275,7 @@ void ex_ni(exarg_T *eap)
 {
   if (!eap->skip)
     eap->errmsg = (char_u *)N_(
-        "E319: Sorry, the command is not available in this version");
+        "E319: The command is not available in this version");
```

Comment:
Corresponding change to `runtime/doc/message.txt`, line 525, is missing.

Do you know how to test these changes? I didn't manage to make messages appear in a different language.